dont_have_vinogret.py

```diff
@@ -0,0 +1,28 @@
+
+import datetime
+import random
+
+
+def random_date_in_range_str(start, end):
+    """
+    The function choose random date in the range between the 2 given dates
+    :param start: start date - String!!!
```

Comment:
Read about type hints and return types, this is not how we document types in python (!!!)